jest.config.js

```diff
@@ -1,5 +1,8 @@
 'use strict'
-
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
aren't you disabling this here https://github.com/mojaloop/thirdparty-scheme-adapter/pull/3/files#diff-e4403a877d80de653400d88d85e4801aR60 so that these wont be needed?

Reply:
thanks! fixed!